portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,68 @@
+      <p>This project was originally suppose to be a side scrolling game, however me and my partner decided against that and made a game where you can select different characters, and battle in a 2D plan, where enemies have basic AI and move around the map. Battles in the game were represented in a Rock, Paper, Scissor system and was shown by basic animation. Some of the things I contributed was making the title screen, character select screen, controls in both of these screen, I also included methods to make the screen full screen automatic and close by it self, which weren't included in the starter code. I also made the 2D animations, and helped with how the AI logic worked. As well as other small things.</p>
+      <p>Some of things I enjoy doing, is playing games, and building legos. I actually really good at building legos that 1000+ piece set I can finish in about 5 hours, given no interuptions and I'm focus. I enjoy looking at New York City skyline. In terms of Lego set I have a small New York City skyline set, which includes Empire State Building, Statue of Liberty, Crystaler Building, Flatiron Building and One World Trade Center, and in addition to that bigger lego models of the Empire State Building and Statue of Liberty. I also generally like model building. I have a plastic model of One World Trade Center in my room and when I was younger I had a paper model of the original and new World Trade Center.</p>
+      <p>Below is a link to github.<br>https://github.com/FunAverage57/</p>
+      <p>Below is just random images of stuff I like or I feel like explain what I am into</p>
+      <p>This is Greninja, one of my favorite Pokemon.</p>
+      <img src = "images/Greninja.png" style="width:1000px;height:600px;"/>
+      <p>This is my older cat Peanut, he passed away March 2020.</p>
+      <img src = "images/IMG_5016.JPG" style="width:1000px;height:600px;"/>
+      <p>This is my younger cat Chocolate, he loves a lot of attention, and this is an adorable picture of him.</p>
+      <img src = "images/IMG_5040.JPG" style="width:1000px;height:600px;"/>
+      <p>This is a picture of the before mention Lego Statue of Liberty model, along with the Lego New York City Skyline set.</p>
+      <img src = "images/IMG_5021.JPG" style="width:1000px;height:1200px;"/>
+      <p>This is a picture of my Nintendo Switch, I spend way too much money to get those joy con colors, which blue and green are my favorite and second favorite colors. Also something is wrong with the image, see if you notice it.</p>
+      <img src = "images/IMG_5031.JPG" style="width:1000px;height:600px;"/>
+      <p>This is a painting done for one of my classes of downtown Manhattan skyline.</p>
+      <img src = "images/IMG_5132.JPG" style="width:1000px;height:600px;"/>
+      
+      <!--<p>Click here to get a random fact or one of the shows I love:</p>
       <button onclick="addRandomGreeting()">Hello</button>
       <div id="greeting-container"></div>
-    </div>
+    </div>-->
```

Comment:
I would avoid leaving commented out code in a commit.

If we want to add it back later, the advantage of git is that it will be in the history!